### PR TITLE
Add variants schema to NL

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

As the title states. A new schema on the national level that describes the different variants and their data.

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/app/schema/nl/variants.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        "theta_is_variant_of_concern",
+        "kappa_percentage",
+        "kappa_occurrence",
+        "kappa_is_variant_of_concern",
+        "other_percentage",
+        "other_occurrence",
+        "other_is_variant_of_concern",
+        "sample_size",
+        "date_start_unix",
+        "date_end_unix"
```
Comment (VWSCoronaDashboard8) [inline packages/app/schema/nl/variants.json:36]: I think we might want to add date_of_insertion here

File: packages/app/schema/nl/variants.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
+{
+  "definitions": {
+    "value": {
+      "title": "national_variants_value",
```
Comment (VWSCoronaDashboard8) [inline packages/app/schema/nl/variants.json:4]:
```suggestion
      "title": "nl_variants_value",
```

File: packages/app/schema/nl/__difference.json [hunk 3]
```diff
@@ -135,12 +158,8 @@
           "type": "integer"
         }
       },
-      "required": [
-        "old_value",
-        "difference",
-        "new_date_unix"
-      ],
+      "required": ["old_value", "difference", "new_date_unix"],
```
Comment (VWSCoronaDashboard8) [inline packages/app/schema/nl/__difference.json:161]: Seems to be mister @VWSCoronaDashboard4 🕵️‍♂️

File: packages/app/schema/nl/variants.json [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+          "type": "integer"
+        },
+        "date_end_unix": {
+          "type": "integer"
+        }
+      }
+    }
+  },
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "national_variants",
```
Comment (VWSCoronaDashboard8) [inline packages/app/schema/nl/variants.json:134]:
```suggestion
  "title": "nl_variants",
```

File: packages/app/schema/nl/variants.json [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+        "theta_occurrence",
+        "theta_is_variant_of_concern",
+        "kappa_percentage",
+        "kappa_occurrence",
+        "kappa_is_variant_of_concern",
+        "other_percentage",
+        "other_occurrence",
+        "other_is_variant_of_concern",
+        "sample_size",
+        "date_of_insertion ",
```
Comment (VWSCoronaDashboard14) [inline packages/app/schema/nl/variants.json:35]: extra space

Comment (VWSCoronaDashboard1) [reply packages/app/schema/nl/variants.json]: oopz

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**